src/main/java/rx/internal/operators/OperatorOnBackpressureBuffer.java

```diff
@@ -71,6 +175,7 @@ public OperatorOnBackpressureBuffer(long capacity, Action0 onOverflow) {
 
         return parent;
     }
+
     private static final class BufferSubscriber<T> extends Subscriber<T> implements BackpressureDrainManager.BackpressureQueueCallback {
```

Comment:
`BufferSubscriber` is private here but the public `BackpressureOverflowStrategy` exposes it, which makes it not extendable outside this package, which is not better than having a `BackpressureOverflowStrategy` enum in the first place. Plus, leaving `BufferSubscriber` private generates accessibility bridges.